### DISCUSSION TITLE
Absolutely impossible to use in case of 50k records in domain.

### DISCUSSION THREAD

**Discussion by hyppopotamus** [ngoduykhanh/PowerDNS-Admin#1650] [Q&A]
I have approximately 50k records in domain A, 8k record in domain B and 6k record in domain C.
So, test showed that gui could render up to 10k records in answer (slow but possible to use domains B and C). But in case domain A - rendering is almost endless.
I could not find any options, that could help me in this case. No "request window size" or something else.
PDNS backend is postgresql.
Need help. Any ideas? (axcept new 100GHz cpu in workstation)

**Comment by subbink** [comment]
Would it be possible to split domain A in smaller parts?
If you have a lot of labels below a certain label, you could create a separate zone/domain for that.
For example if you have a lot of labels under _report._dmarc.example.org, you could create a zone/domain for that name and move all items to that zone.

**Reply by hyppopotamus** [reply]
It is obvious but not suitable. Moreover, domains B and C are growing to 50000 records.
